src/Pos/Web/Server.hs

```diff
@@ -77,12 +77,13 @@ applicationGT = do
     return $ serve gtNodeApi server
 
 -- [CSL-217]: do not hardcode logStdoutDev.
-serveImpl :: MonadIO m => m Application -> String -> Word16 -> m ()
-serveImpl application host port =
-    liftIO . runSettings mySettings . logStdoutDev =<< application
+serveImpl :: MonadIO m => m Application -> String -> Word16 -> FilePath -> FilePath -> FilePath -> m ()
```

Comment:
Many functions take three `FilePath` arguments. It's cumbersome and error-prone (because one can confuse the order for example). I would wrap them into one type. You can also define command-line options parser for this type and use it for both `node` and `lwallet`.
It would probably allow to avoid a bug in `lwallet` options (where two different options have the same name), because there is no bug in `NodeOptions`.